wandb/sdk/data_types/base_types/wb_value.py

```diff
@@ -11,6 +11,31 @@
     TypeMappingType = Dict[str, Type["WBValue"]]
 
 
+def _is_maybe_offline() -> bool:
```

Comment:
i guess it is fine for now since in the default case it will be backward compatible

Reply:
I would also like to follow up here and completely remove this in the next incompatible release.